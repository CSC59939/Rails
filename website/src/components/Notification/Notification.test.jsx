import React from 'react';
import renderer from 'react-test-renderer';
import { Notification } from '..';

describe('Notification', () => {
  it('Render Notification without any eror', () => {
    expect(renderer.create(<Notification />).toJSON()).toMatchSnapshot();
  });
});
