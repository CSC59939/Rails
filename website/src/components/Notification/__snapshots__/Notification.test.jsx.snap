// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification Render Notification without any eror 1`] = `null`;

exports[`ProfileInfo Render ProfileInfo without any eror 1`] = `null`;
